app/controllers/assignments_controller.rb

```diff
@@ -0,0 +1,218 @@
+class AssignmentsController < ApplicationController
+  before_action :set_assignment, only: [:show, :edit, :update, :destroy, :commit, :reject, :review]
+  before_action :set_assignment_and_digital_object_from_new_or_create_params, only: [:new, :create]
+  before_action :require_appropriate_project_permissions!
+  before_action :set_contextual_nav_options
+
+  # GET /assignments
+  def index
+    @assignments = Assignment.where(assignee: current_user)
+    @assignations = Assignment.where(assigner: current_user)
+  end
+
+  # GET /assignments/new
+  def new
+  end
+
+  # POST /assignments
+  def create
+    @assignment.status = 'unassigned' if @assignment.assignee.blank?
+
+    # original and proposed fields should both be set upon assignment creation
+    case @assignment.task
+    # TODO: Add 'describe', 'annotate', and 'sequence' types to case statement
```

Comment:
annotate and describe are here; ticket the sequence assignment and update comment?

Reply:
Updated!